meetup_ballot/ballot.py

```diff
@@ -112,13 +123,19 @@ def filter_spam_members(member_ids, client):
             )
             good_members.append(member_id)
         else:
-            logging.info(
-                "Bad Member name: {} (ID: {})".format(member_name, member_id)
-            )
+            if member_id in read_name_exceptions(name_exceptions_csv):
```

Comment:
I think you want to move the call to `read_name_exceptions` outside the loop, otherwise the file will be re-read on every iteration.

Reply:
Good point! Wasn't sure how to run the code end-to-end, did run the tests and they were passing but I think there is some bug somewhere in how unittest is set up in virtualenv, can we rely on Travis ?

Reply:
What's the alternative you propose when moved outside the loop?

Reply:
Pushed changes, but I need to see how to run the tests properly!